frontend/app/services/personal-information-service.server.ts

```diff
@@ -6,106 +6,113 @@ import { getLogger } from '~/utils/logging.server';
 
 const log = getLogger('personal-information-service.server');
 
-const personalInformationApiResponseSchema = z.object({
-  Client: z.object({
-    ClientIdentification: z
-      .object({
-        IdentificationID: z.string().optional(),
-        IdentificationCategoryText: z.string().optional(),
-      })
-      .array()
-      .optional(),
-    PersonName: z
+const personalInformationApiResponseSchema = z
+  .object({
+    Client: z.object({
+      ClientIdentification: z
+        .object({
+          IdentificationID: z.string().optional(),
+          IdentificationCategoryText: z.string().optional(),
+        })
+        .array()
+        .optional(),
+      PersonName: z
+        .object({
+          PersonSurName: z.string().array().optional(),
+          PersonGivenName: z.string().array().optional(),
+        })
+        .array()
+        .optional(),
+      PersonContactInformation: z
+        .object({
+          EmailAddress: z.object({
+            EmailAddressID: z.string(),
+          }),
+          TelephoneNumber: z.object({
+            FullTelephoneNumber: z.string(),
+          }),
+          Address: z
+            .object({
+              AddressCategoryCode: z.object({
+                ReferenceDataName: z.string(),
+              }),
+              AddressStreet: z.object({
+                StreetName: z.string(),
+              }),
+              AddressSecondaryUnitText: z.string(),
+              AddressCityName: z.string(),
+              AddressProvince: z.object({
+                ProvinceName: z.string(),
+              }),
+              AddressCountry: z.object({
+                CountryName: z.string(),
+                CountryCode: z.string().optional(), //TODO ask if this field is a typo
+              }),
+              AddressPostalCode: z.string(),
+            })
+            .array(),
+        })
+        .optional(),
+      PersonLanguage: z
+        .object({
+          LanguageCode: z.object({
+            ReferenceDataName: z.string(),
+          }),
+          PreferredIndicator: z.boolean(),
+        })
+        .optional(),
+      PersonSINIdentification: z.object({
+        IdentificationID: z.string(),
+      }),
+    }),
+  })
+  .optional();
```

Comment:
Is there any reason why the entire API response object can be optional? We should either expect a response body represented as a personal info object back with a 200, or a 204 where we wouldn't parse the response body with this schema.

Reply:
I was doing it for another error but it looks like it wasn't needed. Fixed 102180defa666472647423336ecff5b8c2e4f0c2